package.json

```diff
@@ -81,16 +81,16 @@
     "comlink": "^4.4.1",
     "codemirror": "^5.50.2",
     "decko": "^1.2.0",
-    "htm": "^2.2.1",
+    "htm": "^3.1.1",
     "linkstate": "^1.1.1",
     "magic-string": "^0.25.7",
     "marked": "^0.8.0",
     "node-fetch": "^2.6.1",
-    "preact": "^10.4.4",
-    "preact-custom-element": "^4.0.0",
-    "preact-markup": "^2.0.0",
+    "preact": "10.15.1",
```

Comment:
Locked to `10.15.1` is intentional for the moment. `preact-iso`'s test suite seemingly caught some regression in Preact that I haven't yet been able to look into, and as I'm trying to migrate this repo over to iso w/ vite, locking to the known working version is simply easier.